torchgeo/models/swin.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Pre-trained Swin v2 Transformer models."""
+
+from typing import Any, Optional
+
+import kornia.augmentation as K
+import torch
+import torchvision
+from torchvision.models._api import Weights, WeightsEnum
+from torchvision.models.swin_transformer import SwinTransformer
```

Comment:
```suggestion
from torchvision.models import SwinTransformer
```
The sub-sub-modules aren't really public.